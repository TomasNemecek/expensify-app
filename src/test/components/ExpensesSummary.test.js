import React from 'react';
import {shallow} from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('Should render Expenses summary correctly with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>)
    expect(wrapper).toMatchSnapshot();
});

test('Should render ExpensesSummary with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={24} expensesTotal={134155225}/>)
    expect(wrapper).toMatchSnapshot();
});