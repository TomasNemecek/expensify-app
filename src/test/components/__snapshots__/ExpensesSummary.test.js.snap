// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Expenses summary correctly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`Should render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $1,341,552.25
  </h1>
</div>
`;
